.github/workflows/monitor-deployment.yml

```
@@ -0,0 +1,82 @@
+    
+    steps:
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ env.AWS_REGION }}
+        
+    - name: Get Environment URL
+      id: get-url
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:5:27: Double quote to prevent globbing and word splitting

(shellcheck)